src/main/java/com/ecommerce/ECommerceApp/model/AccountType.java

```diff
@@ -1,5 +1,9 @@
 package com.ecommerce.ECommerceApp.model;
 
+/**
+ * Enumerator containing different types of user account types
+ * Includes: USER, SUPPLIER_USER, SUPPLIER_ADMIN, ADMIN
+ */
```

Comment:
Added missing Javadoc to help clarify purpose of code, keeping it maintainable and readable.

Source: https://www.tutorialspoint.com/java/java_documentation.htm